src/main/blackbox/blackbox.c

```diff
@@ -1675,10 +1671,25 @@ void blackboxUpdate(timeUs_t currentTimeUs)
  */
 void blackboxInit(void)
 {
+    blackboxResetIterationTimers();
+
+    // an I-frame is written every 32ms
+    // gyro.targetLooptime is 1000 for 1kHz loop, 500 for 2kHz loop etc, gyro.targetLooptime is rounded for short looptimes
+    if (gyro.targetLooptime == 31) { // rounded from 31.25us
```

Comment:
`blackboxIInterval` is monotonic even with special cases

targetLooptime  | 64 | 63 | 62 | .. | 32 | 31 | 30
--- | --- | --- | --- | --- | --- | ---  | ---
I-interval | 500 | 507 | 516 | .. | 1000 | 1032 | 1066
with correction | 500 | 512 | 516 | .. | 1000 | 1024 | 1066